rs/rosetta-api/ledger_canister_blocks_synchronizer/src/blocks.rs

```diff
@@ -802,39 +730,30 @@ impl Blocks {
         tx.execute(
             r#"
             CREATE TABLE IF NOT EXISTS blocks (
-                hash BLOB NOT NULL,
-                block BLOB NOT NULL,
+                block_hash BLOB NOT NULL,
+                encoded_block BLOB NOT NULL,
                 parent_hash BLOB,
-                idx INTEGER NOT NULL PRIMARY KEY,
+                block_idx INTEGER NOT NULL PRIMARY KEY,
                 verified BOOLEAN,
-                timestamp TEXT
-            )
-            "#,
-            [],
-        )?;
-        tx.execute(
-            r#"
-            CREATE TABLE IF NOT EXISTS transactions (
-                block_idx INTEGER NOT NULL,
+                timestamp TEXT,
                 tx_hash BLOB NOT NULL,
                 operation_type VARCHAR NOT NULL,
                 from_account VARCHAR(64),
                 to_account VARCHAR(64),
                 spender_account VARCHAR(64),
-                amount INTEGER,
+                amount TEXT,
```

Comment:
I was curious about this change of the type from `INTEGER` to `TEXT` (same for `fee` below), since as `TEXT`, it doesn't seem like we can perform any calculations on the value, but it looks like this field is not for anything like that? Also, in e.g., `get_transaction`, we parse the transaction from `encoded_block`, so it's not used there either. Is this field mainly for debugging, or what is the purpose?